webpack.config.dev.js

```diff
@@ -0,0 +1,22 @@
+const merge = require('webpack-merge');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+let dev = {
```

Comment:
const are not exactly constants on JavaScript, I and a lot of people use it only for require o very specific things :)